.github/workflows/deploy-docs.yml

```diff
@@ -22,6 +22,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Get TAG
+        run: echo "TAG=${{ github.ref_name }}" >> $GITHUB_ENV
+
+      - name: Get DOC_DIR
+        run: |
+          DOC_DIR="$(cut -d '.' -f 1,2 <<< "$TAG")"
+          echo "DOC_DIR=${DOC_DIR:1}".x >> $GITHUB_ENV
```

Comment:
couldn't figure out how to make this a one liner, suggestions welcome. Works as is and outputs e.g., "0.1.x"